lib/src/detail/detail_page.dart

```diff
@@ -31,8 +37,9 @@ class DetailPage extends ConsumerWidget {
   Widget build(BuildContext context, WidgetRef ref) {
     final snapModel = ref.watch(snapModelProvider(snapName));
     final updatesModel = ref.watch(updatesModelProvider);
+
     return snapModel.state.when(
-      data: (_) => ResponsiveLayoutBuilder(
+      data: (snapData) => ResponsiveLayoutBuilder(
```

Comment:
`state` is `AsyncValue<void>`, so the argument of the closure doesn't need to be named
```suggestion
      data: (_) => ResponsiveLayoutBuilder(
```